x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/add_source/byo_source_prompt.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues } from 'kea';
+
+import {
+  EuiEmptyPrompt,
+  EuiImage,
+  EuiLink,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiSpacer,
+} from '@elastic/eui';
+
+import { EuiButtonTo } from '../../../../../shared/react_router_helpers';
+
+import { getSourcesPath, getAddPath } from '../../../../routes';
+
+import { SourcesLogic } from '../../sources_logic';
+
+import illustration from './illustration.svg';
+
+export const BYOSourcePrompt: React.FC = () => {
+  const { externalConfigured } = useValues(SourcesLogic);
+
+  return (
+    <EuiEmptyPrompt
```

Comment:
This whole file needs i18n, I'm holding off until at least the first round of UX feedback